src/actions/wallet/L1/writeSendMessage.test.ts

```diff
@@ -0,0 +1,64 @@
+import { accounts } from '../../../_test/constants'
+import { publicClient, testClient, walletClient } from '../../../_test/utils'
+import { writeSendMessage } from './writeSendMessage'
+import { l1CrossDomainMessengerABI } from '@eth-optimism/contracts-ts'
+import { Hex, decodeEventLog, encodeFunctionData } from 'viem'
+import { mine } from 'viem/actions'
+import { base } from 'viem/chains'
+import { expect, test } from 'vitest'
+
+test('can successfully call sendsMessage', async () => {
```

Comment:
Bless your heart writing tests for a library that eventually will get rewritten :)

Reply:
can't ship untested code!

Reply:
p good but we can improve 
<img width="748" alt="image" src="https://github.com/base-org/op-viem/assets/6678357/fdffc7ac-863d-4ec5-b523-e59d24e05278">

Reply:
This is amazing test coverage! The actions are simple enough we can definitely get to 100% after API is stable